src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -516,6 +518,8 @@ const requestToState = (composeRequest, distroInfo, isProd, enableOscap) => {
     if (enableOscap) {
       formState['oscap-profile'] =
         composeRequest?.customizations?.openscap?.profile_id;
+      composeRequest?.customizations?.openscap?.profile_name;
+      composeRequest?.customizations?.openscap?.profile_description;
```

Comment:
Same here, I'd expect something like:
```suggestion
      formState['oscap-profile-description'] = composeRequest?.customizations?.openscap?.profile_description;
```